wiki/People/Quality_Assurance_Team/en.md

```diff
@@ -4,11 +4,11 @@
 Quality Assurance Team
 ======================
 
-The **Quality Assurance Team**, commonly referred to as QAT, are the judges of quality control who will check recently qualified beatmaps for quality and playability, retracting their qualified status as a disqualification notice where appropriate. They form the bulwark of standard control and enforce the basic expectation of quality for all beatmaps that enter the ranking process.
+The Quality Assurance Team, commonly referred to as QAT, form the last line of defense for standard control and enforce the basic expectation of quality for all beatmaps that enter the ranking process. Along with the [Quality Assurance Helpers](https://osu.ppy.sh/community/forums/topics/764134), they check recently Qualified beatmaps for any issues that may have slipped past the nomination and modding processes. Mapsets that contain issues have their Qualified status revoked through disqualification where necessary.
```

Comment:
don't link a forum thread that seems like it should be its own article? If you think that information is important enough to be linked on the wiki make it a wiki article

Reply:
made it link to a subheader on the bn page